schema_based_on_diagram.sql

```diff
@@ -0,0 +1,56 @@
+/* Database schema to keep the structure of the entire database. */
+
+CREATE TABLE patients (
+    id serial PRIMARY KEY,
+    name varchar(255),
+    date_of_birth date,
+);
+
+CREATE TABLE treatment (
+    id serial PRIMARY KEY,
+    name varchar(255),
+    type varchar(255),
+);
+
+CREATE TABLE medical_histories (
+    id serial PRIMARY KEY,
+    addmited_at timestamp,
+    patient_id integer REFERENCES patients(id),
+    status varchar(255),
+);
```

Comment:
- [ ] Please use the `CREATE INDEX` statement to explicitly create indexes on the foreign key columns in your respective tables. These indexes can help improve query performance when you're searching or joining data using these foreign key columns. See examples below 👇 
```SQL
CREATE INDEX ON medical_histories (patient_id);
CREATE INDEX ON invoices (medical_history_id);
```